src/components/webView.re

```diff
@@ -119,12 +119,62 @@ let make =
             "injectedJavaScript": fromOption(injectedJavaScript),
             "mediaPlaybackRequiresUserAction":
               fromOption(mediaPlaybackRequiresUserAction),
-            "onError": fromOption(onError),
-            "onLoad": fromOption(onLoad),
-            "onLoadEnd": fromOption(onLoadEnd),
-            "onLoadStart": fromOption(onLoadStart),
+            "onError": fromOption(UtilsRN.option_map((callback) => {
```

Comment:
This is repeated multiple times. Could you factor it out into a utility function?